Update install instructions (#59)
ignitionrobotics/sdf_tutorials: install/tutorial_9_later.md

    wget http://packages.osrfoundation.org/gazebo.key -O - | sudo apt-key add -
    sudo apt-get update

    sudo apt install libsdformat<#>-dev libsdformat<#>

Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
which version you need.

### macOS

On macOS, add OSRF packages:

    ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
    brew tap osrf/simulation

Install sdformat:

    brew install sdformat<#>

Be sure to replace `<#>` with a number value, such as 1 or 2, depending on
which version you need.

### Windows

Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
Miniconda suffices.

Create if necessary, and activate a Conda environment:

    conda create -n ign-ws
    conda activate ign-ws

Install `sdformat`:

    conda install libsdformat --channel conda-forge

You can view all the versions with

    conda search libsdformat --channel conda-forge

and install a specific minor version with

    conda install libsdformat=9.3.0 --channel conda-forge

> ## Source installation from source

Reviewer: ```suggestion
## Source installation
```

### Ubuntu

#### Prerequisites

Make sure you have removed the Ubuntu pre-compiled binaries before installing from source

    sudo apt-get remove 'libsdformat*' 'sdformat*'

If you have previously installed from source, be sure you are installing to the same path location or that you have removed the previous installation from source version manually.

#### Install Required Dependencies

Install prerequisites. A clean Ubuntu system will need:

##### build tools, ruby for building xml schemas, tinyxml, and boost:

>     sudo apt-get install ruby-dev build-essential libtinyxml2-dev cmake pkg-config

Reviewer: There are missing dependencies, I recommend using `packages.apt` instead of listing the dependencies here

by downstream software.
This may be suitable for use inside a docker container,
but care should be taken when installing to `/usr` since installing the
debian packages would overwrite the version that you built from source,
and it can be very difficult to debug problems that arise from this system state.
It is often safer to install to a local folder and set the `LD_LIBRARY_PATH`
and other environment variables appropriately.
To simplify this type of installation, a
[catkin workspace](http://gazebosim.org/tutorials?tut=install_from_source&cat=install#Installinacatkinworkspace)
or
[colcon workspace](https://index.ros.org/doc/ros2/Tutorials/Colcon-Tutorial/)
may be used along with the corresponding build tools that generate
shell scripts for setting the necessary environment variables.

#### Installing into `/usr`

1. Clone the repository into a directory in your home folder:

        mkdir ~/sdf_source
        cd ~/sdf_source/
        git clone https://github.com/ignitionrobotics/sdformat

2. Change directory into the sdformat repository and switch to the sdf<#> branch

        cd sdformat
        git checkout sdf<#>

>    **Note: the <tt>main</tt> branch is the development branch where you'll find the bleeding edge code, your cloned repository should be on this branch by default but we recommend you switch to the default branch if you desire more stability**

Reviewer 2: Since the default branch is now the release/stable branch, I recommend rewording this as such:

```suggestion
   **Note: Your cloned repository should be on the latest release branch by default, which is the recommended branch for stability. You can find the bleeding edge code in the <tt>main</tt> branch.**
```
